client/src/main/java/org/apache/iotdb/client/AbstractClient.java

```diff
@@ -221,6 +221,7 @@ private static void output(ResultSet res, boolean printToConsole, ZoneId zoneId)
       }
       if (displayCnt == maxPrintRowCount) {
         println(String.format("Reach maxPrintRowCount = %s lines", maxPrintRowCount));
+        return;
```

Comment:
I suggest you remove the `displayCnt` because it is now useless.
And you can eliminate this check statement (`if(displayCnt == maxPrintRowCount) ...`) and check out of this loop by:
```
if(!res.next()) {
    printCount(isShow, res, cnt);
}
else {
    println(String.format("Reach maxPrintRowCount = %s lines", maxPrintRowCount));
} 
```

Reply:
I don't agree you,  because data still exists in `res.next()` , we just stopped getting data because we reached the `maxPrintRowCount `